deployment/clouddeploy/gke-workers/environments/oss-vdb-test/kustomization.yaml

```diff
@@ -13,3 +13,4 @@ patches:
 - path: gen-cperepos-map.yaml
 - path: nvd-cve-osv.yaml
 - path: alias-computation.yaml
+- path: backup.yaml
```

Comment:
This will have to go under the `resources` section at the top, not the `patches` section